fmriprep/interfaces/reports.py

```diff
@@ -34,6 +34,7 @@
 
 FUNCTIONAL_TEMPLATE = """\t\t<h3 class="elem-title">Summary</h3>
 \t\t<ul class="elem-desc">
+\t\t\t<li>Repetition Time: {tr}</li>
```

Comment:
```suggestion
\t\t\t<li>Repetition time (TR): {tr:.03g}s</li>
```

`{:.03g}` formats floats to up to three decimals, but drops trailing zeros. And we should use units.